src/solvers/indicators_1d.jl

```diff
@@ -0,0 +1,115 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+# Modified indicator for ShallowWaterEquations1D to apply full FV method on cells
```

Comment:
```suggestion
# Modified indicator for ShallowWaterEquationsWetDry1D to apply full FV method on cells
```

Reply:
Done in 77f44c650f95e4bc7bcbd284ecd2f60f6b9d0609